apps/web/app/(navigation)/CreateNewBudget.tsx

```diff
@@ -118,6 +121,55 @@ export const CreateNewBudget = ({ onClose }: NewBudget) => {
     currentLang === "pl" && setLang("pl-PL");
   }, [lang, currentLang]);
 
+  // TRANSFORM TIMESTAMP TO ISO (+7200000 miliseconds, unix timestamp is in seconds, but Javascript counts in miliseconds, * 1000 not work)
+  const startDateTimestamp = newBudget.dateStart + 7200000;
+  const endDateTimeStamp = newBudget.dateEnd + 7200000;
+  const budgetStartDate = new Date(startDateTimestamp).toISOString();
+  const budgetEndDate = new Date(endDateTimeStamp).toISOString();
+
+  const url = "https://inbudget-patronage-api-dev.azurewebsites.net/budgets";
+
+  const token =
+    "eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJha0lYQnV6SHhGb1RINkgxRFNhTkRiVlk4MnBMWXRNdFdVMkRPTjNHTXNnIn0.eyJleHAiOjE2ODM1ODQ2NTgsImlhdCI6MTY4MzU3NzQ1OCwianRpIjoiOTliN2JkOTItNjRmNS00NDQwLTk4YmEtZDlhMjUxYWI1MTIzIiwiaXNzIjoiaHR0cHM6Ly9rZXljbG9hay1pbmJ1ZGdldC1wYXRyb25hZ2UyMDIzLmF6dXJld2Vic2l0ZXMubmV0L3JlYWxtcy9pbmJ1ZGdldC1yZWFsbS1kZXYiLCJhdWQiOiJhY2NvdW50Iiwic3ViIjoiODIwNjNmMmUtOWQ5YS00YjM4LWEyMmUtNTU3MmNlZTlkZGY0IiwidHlwIjoiQmVhcmVyIiwiYXpwIjoiaW5idWRnZXQtY2xpZW50Iiwic2Vzc2lvbl9zdGF0ZSI6IjA2ZDliY2FhLTZjOTEtNDRlZS1hMTJiLTNiMDIzNWIxMDNjZSIsImFjciI6IjEiLCJhbGxvd2VkLW9yaWdpbnMiOlsiLyoiXSwicmVhbG1fYWNjZXNzIjp7InJvbGVzIjpbIm9mZmxpbmVfYWNjZXNzIiwiZGVmYXVsdC1yb2xlcy1pbmJ1ZGdldC1yZWFsbS1kZXYiLCJ1bWFfYXV0aG9yaXphdGlvbiJdfSwicmVzb3VyY2VfYWNjZXNzIjp7ImFjY291bnQiOnsicm9sZXMiOlsibWFuYWdlLWFjY291bnQiLCJtYW5hZ2UtYWNjb3VudC1saW5rcyIsInZpZXctcHJvZmlsZSJdfX0sInNjb3BlIjoicHJvZmlsZSBlbWFpbCIsInNpZCI6IjA2ZDliY2FhLTZjOTEtNDRlZS1hMTJiLTNiMDIzNWIxMDNjZSIsImVtYWlsX3ZlcmlmaWVkIjp0cnVlLCJuYW1lIjoic211dG5hIHphYmEiLCJhdmF0YXIiOiIxIiwicHJlZmVycmVkX3VzZXJuYW1lIjoic211dG5hcnphYmFAcG5nLnBsIiwiZ2l2ZW5fbmFtZSI6InNtdXRuYSIsImZhbWlseV9uYW1lIjoiemFiYSIsImVtYWlsIjoic211dG5hcnphYmFAcG5nLnBsIn0.N0qloqsbooU95-WjOsG2uDChJnz6TD6nBlT1inIeMIQXhulYBYhwJixFCGs2bgZqeAZ5JvwmQM9mE575DQ4bDSvcPvDFrG3ffg6qcMGDVaZq6invYE5xXcmCyAsoRSfmSsSupN8ASaD1ShVBR2_UelsIf8-4FsOMjHgKBHj7ZgoRDAdkLCRDiNmLwTn_KE27QOO6Pj8wOIhr6Vht5QUVtvNV0dKc0CApvxoFOZoBb_Ljr8WInC2aLUUQ_6RZI0JoOqSUNPWIRjMmMycfM3tTtl_esNfh-L8YcUv7-2_a_jGTPQGYf2iVG4yGGZ7gBgDPouYgJqD6C91PqQ6W1qcSjQ";
+
+  const queryClient = useQueryClient();
+
+  const useSendBudget = () =>
+    useMutation(
+      () =>
+        fetch(url, {
+          method: "POST",
+          headers: {
+            accept: "text/plain",
+            Authorization: "Bearer " + token,
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({
+            id: `${uuidv1()}`,
+            name: `${newBudget.name}`,
+            userId: `${uuidv1()}`,
+            limit: {
+              value: `${newBudget.limit}`,
+              currency: `${newBudget.currency.tag}`,
+            },
+            period: {
+              startDate: `${budgetStartDate}`,
+              endDate: `${budgetEndDate}`,
+            },
+            description: `${newBudget.description}`,
+            iconName: `${newBudget.icon}`,
+          }),
```

Comment:
You are using string interpolation on every value here, while everything, except the value, is a string already. And value should be sent to the backend as a number, so also not needed here.

```suggestion
          body: JSON.stringify({
            id: uuidv1(),
            name: newBudget.name,
            userId: uuidv1(),
            limit: {
              value: newBudget.limit,
              currency: newBudget.currency.tag,
            },
            period: {
              startDate: budgetStartDate,
              endDate: budgetEndDate,
            },
            description: newBudget.description,
            iconName: newBudget.icon,
          }),
```

Reply:
Done :)